base/server/upgrade/10.10.0/03-RemoveLegacyTomcatJSSConfig.py

```diff
@@ -0,0 +1,43 @@
+# Authors:
+#     Alexander Scheel <ascheel@redhat.com>
+#
+# Copyright Red Hat, Inc.
+#
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+from __future__ import absolute_import
+import logging
+from lxml import etree
+
+import pki
+
+logger = logging.getLogger(__name__)
+
+
+class RemoveLegacyTomcatJSSConfig(pki.server.upgrade.PKIServerUpgradeScriptlet):
+
+    def __init__(self):
+        super(RemoveLegacyTomcatJSSConfig, self).__init__()
+        self.message = 'Remove legacy TomcatJSS Configuration from server.xml'
+
+        self.parser = etree.XMLParser(remove_blank_text=True)
+
+    def upgrade_instance(self, instance):
+        logger.info('Upgrading %s', instance.server_xml)
+        self.backup(instance.server_xml)
+
+        document = etree.parse(instance.server_xml, self.parser)
+
+        server = document.getroot()
+
+        removed_attrs = ['strictCiphers', 'sslVersionRangeStream', 'sslVersionRangeDatagram',
+                         'sslRangeCiphers']
+
+        connectors = server.findall('Service/Connector')
+        for connector in connectors:
+            for attr in removed_attrs:
+                if connector.get(attr):
+                    connector.attrib.pop(attr, None)
```

Comment:
No big deal, but maybe we can skip the `get()` in line 39 and call `pop()` directly?

Reply:
Yep, that should work.